### ISSUE TITLE
[BUG] Using Intents.ALL hangs the bot, fails to send heartbeat

### ISSUE THREAD

**Issue by V3ntus** [interactions-py/interactions.py#462]
**Describe the bug**
On `unstable` branch, according to commit [e836e50](https://github.com/interactions-py/library/commit/e836e50f727ee91f81274b8b775f39530f72b166), passing `interactions.Intents.ALL` into `interactions.Client` results in a log message `DEBUG:gateway:The heartbeat took too long to send.`

**Reproducing steps**
```py
bot = interactions.Client(
    token=TOKEN,
    disable_sync=True,  # False also gives the same issue
    intents=Intents.ALL
)
```

**What's normally expected**
Send a heartbeat; Respond to commands

**What actually happened**

```py
--- Logging error ---
Traceback (most recent call last):
  File "/Users/ventus/GitRepos/ModerationBot/.venv/lib/python3.9/site-packages/interactions/api/gateway.py", line 57, in run
    coro.result(timeout=10)
  File "/Users/ventus/development/miniconda3/lib/python3.9/concurrent/futures/_base.py", line 447, in result
    raise TimeoutError()
concurrent.futures._base.TimeoutError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/ventus/development/miniconda3/lib/python3.9/logging/__init__.py", line 1083, in emit
    msg = self.format(record)
  File "/Users/ventus/development/miniconda3/lib/python3.9/logging/__init__.py", line 927, in format
    return fmt.format(record)
TypeError: format() missing 1 required positional argument: 'record'
Call stack:
  File "/Users/ventus/development/miniconda3/lib/python3.9/threading.py", line 912, in _bootstrap
    self._bootstrap_inner()
  File "/Users/ventus/development/miniconda3/lib/python3.9/threading.py", line 954, in _bootstrap_inner
    self.run()
  File "/Users/ventus/GitRepos/ModerationBot/.venv/lib/python3.9/site-packages/interactions/api/gateway.py", line 61, in run
    log.error(
Message: 'The client was unable to send a heartbeat, closing the connection.'
Arguments: ()
ERROR:gateway:The client was unable to send a heartbeat, closing the connection.
```

**Versions**
- [ ] I am using discord.py versions 1.7 and below with my code.
    - [ ] I am using 2.0 or higher, or a modified fork.
- [ ] I am using dis-snek with my code.
- [x] I am not using any of the listed above and am using the library code alone.

**Comment by V3ntus** [comment]
This seems to occur if the bot application only receives partial intent privilege grants. Enabling all gateway intent privileges fixes this issue

**Comment by sillyElvy** [comment]
Try a newer commit, I believe delta commited something with intents

**Comment by V3ntus** [comment]
Oops messed up my commit hash. c0fd655. Latest unstable

**Comment by FayeDel** [comment]
Hiya!
I'm closing this because this is resolved in commit hash e9e721a and this has been fully tested :)

**Comment by V3ntus** [comment]
Awesome, thanks!
